util/chunk/mutrow.go

```diff
@@ -38,6 +38,18 @@ func (mr MutRow) Len() int {
 	return len(mr.c.columns)
 }
 
+// Clone deep clone a MutRow.
+func (mr MutRow) Clone() MutRow {
```

Comment:
Is it better to call [`Row.CopyConstruct`](https://github.com/pingcap/tidb/blob/master/util/chunk/row.go#L211) here?

Reply:
If mr.c == nil, Row.CopyConstruct will panic.